src/app/common/stores/applications/SsvWeb/Operator.store.ts

```diff
@@ -258,6 +258,11 @@ class OperatorStore extends BaseStore {
     this.clearOperatorData();
   }
 
+  getOperatorId() {
```

Comment:
check how getSelectedOperatorsFee is declared and used - do the same with this new field

Reply:
V